packages/voila/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@voila-dashboards/voila",
   "private": true,
-  "version": "1.0.0",
+  "version": "0.2.11",
```

Comment:
For now this package is still private, but will probably be released too when #846 lands